### PR TITLE
Several fixes for new dashboard (v2)

### LINKED ISSUE #2243: Review texts for empty results
Unfortunately there are several scenarios when there isn't any item, for example:

screen shot 2015-02-16 at 10 12 35

screen shot 2015-02-16 at 10 40 43

Some nice texts will be awesome, what do you think?

cc @andrewxhill

### LINKED ISSUE #2360: Decide if load Lato font from Google servers
Need to have the font in CartoDB taking into account offline and onpremise options.

### LINKED ISSUE #2759: Clicking anywhere on the welcome modal should close it

### LINKED ISSUE #2854: Review new dashboard under IE11

### LINKED ISSUE #2862: Button Upload->'Select a file' doesn't appear in Firefox and low screen resolution
Using Firefox 36.0.1 and OS X Yosemite 10.10. 
Screen resolution:**1280x800** (laptop default)

Steps to reproduce:
1. Go to your datasets dashboard.
2. Click on *Connect dataset*

Problem: 
1. You can't scroll down and select Upload a file **Select a file**

Screenshot:
screen shot 2015-03-23 at 13 53 01

Gif:
firefox_connect_dataset

**Note**: It's working correctly with Chrome and Safari. Also it works OK if you open Firefox with a higher ( >1280x800 ) resolution

### PR DESCRIPTION
**Pull request by xavijam**
- [x] Empty results text for liked section (fixes #2243).
- [x] Firefox problems in new create dialog (fixes #2862).
- [x] Clicking anywhere on the welcome modal should close it (fixes #2759).
- [x] IE11 styles reviewed (fixes #2854).
- [x] Remove SEGOE fonts.
- [x] Add Lato fonts (fixes #2360).

### CODE REVIEW

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2473/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2474/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2475/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2476/))

File: app/assets/stylesheets/new_variables/mixins.css.scss [hunk 1]
```diff
@@ -74,31 +74,28 @@
...
 @mixin justify-content($v, $ie-v: "") {
-  $ie-v: $v !default;
   -webkit-justify-content: $v;
   -moz-justify-content: $v;
   -ms-justify-content: $v;
   justify-content: $v;
   // Hello IE10
-  -ms-flex-pack: $ie-v;
+  -ms-flex-pack: if($ie-v != "", $ie-v, $v);
 }
```
Comment (xavijam) [inline app/assets/stylesheets/new_variables/mixins.css.scss]: What do you think about this option?

**Comment by xavijam** [comment]
Not a big deal @viddo, but could you check at least the mixin change?

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2478/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2483/))

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2484/))

File: app/assets/stylesheets/new_variables/mixins.css.scss (hunk 1 above)
Comment (viddo) [inline app/assets/stylesheets/new_variables/mixins.css.scss]: isn't that the same thing? :)

File: app/views/admin/client_applications/new_api_key.html.erb [hunk 2]
```diff
@@ -10,7 +10,6 @@
   <%= javascript_include_tag 'cdb.js', 'models.js', 'new_keys_templates.js', 'new_keys_deps.js', 'new_keys.js' -%>
 <% end %>
 <%= content_for(:css) do %>
-  <%= render 'shared/lato_font' %>
   <%= stylesheet_link_tag 'cartodb.css', 'new_common.css', 'new_keys.css', :media => 'all' %>
```
Comment (viddo) [inline app/views/admin/client_applications/new_api_key.html.erb]: While at it, why not move the common CSS files (cartodb.css and new_common.css) to the [new application layout](https://github.com/CartoDB/cartodb/blob/master/app/views/layouts/new_application.html.erb)?

**Comment by viddo** [comment]
:+1:

File: app/views/admin/client_applications/new_api_key.html.erb (hunk 2 above)
Comment (xavijam) [inline app/views/admin/client_applications/new_api_key.html.erb]: yeah!

Comment (xavijam) [inline app/views/admin/client_applications/new_api_key.html.erb]: Changed done!

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/2490/))

**Pull request merged**